app/services/workflow_parser.rb

```diff
@@ -28,13 +28,19 @@ def create_workflow_steps
     end
   end
 
```

Comment:
Why does this become public?

Reply:
From a prior tweak that I backed away from. Will revert. Good eye!